frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsForm.test.tsx

```diff
@@ -193,7 +192,7 @@ describe('UpdatePropertyDetailsForm component', () => {
   ) => {
     const utils = render(
       <Formik onSubmit={onSubmit} initialValues={renderOptions.initialValues}>
-        {formikProps => <UpdatePropertyDetailsForm {...formikProps} onCancel={onCancel} />}
+        {formikProps => <UpdatePropertyDetailsForm formikProps={formikProps} />}
```

Comment:
If the tests are passing this is probably fine, but this passes the same object instance, so if multiple tests are using that instance and a test modifies the object this _may_ be a problem.

Reply:
That's interesting. Will keep an eye in the future for tests failures